boolean-and-condition.go

```diff
@@ -2,20 +2,34 @@ package main
 
 import "fmt"
 
-func main () {
-  single := true
+func main() {
+	single := true
 
-  if (single) {
-    fmt.Println("Trojan is Single")
-  } else {
-    fmt.Println("Trojan is not single")
-  }
+	if single {
+		fmt.Println("Trojan is Single")
+	} else {
+		fmt.Println("Trojan is not single")
+	}
 
-  employed := false
+	employed := false
 
-  if (employed) {
-    fmt.Println("Kofo is Employed")
-  } else {
-    fmt.Println("Kofo is Unemployed and Jobless at the same time")
-  }
+	if employed {
+		fmt.Println("Kofo is Employed")
+	} else {
+		fmt.Println("Kofo is Unemployed and Jobless at the same time")
```

Comment:
We should remove Trojan & Kofo. Use something more Generic... 

Like `subject`